code/datums/components/crafting/tailoring.dm

```diff
@@ -312,32 +312,32 @@
 
 /datum/crafting_recipe/rainbowbunchcrown
 	name = "Rainbow Flower Crown"
-	result = /obj/item/clothing/head/rainbowbunchcrown/
+	result = /obj/item/clothing/head/flowercrown/rainbowbunch/
```

Comment:
favor, please remove trailing `/` on all these result things. they're very bad

Reply:
There we go, removed.